### ISSUE TITLE
Ethical framework? Or meaningless virtue signaling?

### ISSUE THREAD

**Issue by DrAlta** [opencollective/opencollective#5518]
TL;DR
If business concerns trump our ethical framework. e.g. Stripe requiring use to violate our community guidelines, or they will drop us.

In a world that too frequently feels cruel and hostile, with so many engaged in a dog-eat-dog struggle, why do certain people perform heroic acts of altruism and compassion while others do nothing--or even contribute to the misery of their fellow man?

Though it can be difficult to offer a precise definition of Those People, we can usually identify an example of one when we hear it. Consider the story a Serb woman from the Croatian town of Novska shared with me. In October of 1991, she and her husband fled to the region of Baranja, on the border between Croatia and Serbia, where they lived as refugees. Assuming that it would be safer for him, they sent their 15-year-old son to live with an aunt in the Bosnian town of Zenica. Then they realized that the war was about to break out in Bosnia and Herzegovina as well.

The woman set out for Zenica, determined to bring her son back. She hitchhiked to within 60 miles of the town, but no buses or taxis would go any further.

Desperate, She sat down on the sidewalk and burst into tears.
A stranger came over, a middle-aged man, and asked, “What's wrong, ma'am? May I be of help in some way?”

She told him the reason for her journey, looking up with hope.

“Well, I have to say, ma'am, that you really are asking the impossible!

There are barricades up every few miles from here to Zenica and there are armed men at every checkpoint. That is why no one dares to drive you.” And he walked slowly away, his head bowed.

For at least half an hour she was sitting there, sobbing miserably, not knowing what to do.
Just then a car pulled up in front of her and at the wheel was that same stranger, calling, “Ma'am, get in. I'll do what I can to help you. I'll try to get us over special transport roads that run through the woods. Maybe they don't have barricades up in there yet.”

Early that evening they arrived at her sister's house. Overjoyed, She held her son close and they talked non-stop in the car while the stranger drove without saying a word.
Intoxicated by the delight of being reunited with her son, she was startled when the man said, “Ma'am, this is as far as I can go. The Serbian barricades are just beyond the next curve in the road. You'll have to do that part on foot.”
On her way out of the car she thanked him, and asked what she owed him.
“Nothing, ma'am. Your happiness and your son's are all the payment I need.”

It was only much later that She put two and two together and realized that the stranger had had to stop before the barricades because he was not a Serb.

As this example of human goodness illustrates, Those people actively confronts the choice of whether to defy immorality or keep quiet and accept things the way they are. As Hannah Arendt has powerfully said about humanity: “It is always possible to say ‘yes’ or ‘no,’” and Those People are exactly those who want to make decisions about when to say “no” to evil. The driver in this story was willing to sacrifice his life when he said “no” to the unjust boundaries drawn along constructed ethnic lines and chose to recognize the love of a mother for her son.

The civil courage that characterizes one of Those People is, in the words of the founder and first president of Oxford University's Templeton College, Uwe Kitzinger, “the courage of the non-conformist.”
It is the courage that risks social disapproval, the capacity to resist by thinking critically with one's own mind, and the will to be an active participant in life, not a passive bystander.

Regardless of their differences in age, gender, literacy, religious affiliation, ethnic identity, or wartime roles, Those People share the bravery to risk their lives rather than commit or be complicit in a crime. When so many other people choose to comprise their morals in order to survive, Those Peoples' actions suggest that we must not allow ourselves to be debased by circumstance: To retain our dignity, we must sometimes refuse to live life at any cost.

What sets someone apart and makes them say “no” to evil?

To answer that question, I returned to my World War II rescuers and compared them with people who were bystanders or even Nazi supporters during the war. I found clear differences in worldview, self-image, and the way the different groups classified themselves in relation to others.

Rescuers have self-images that are inclusive and broadly based, with a strong sense of agency; bystanders see themselves as people who are weak on agency, with little control over their lives and little they can do to affect outside events. They think in terms of group identity more than do rescuers, seeing themselves as members of exclusive groups while rescuers see themselves as members of a common humanity.

Finally and ironically, Nazis have a victim mentality, seeing themselves as members of a group that had been treated badly and threatened by Jews, Social Democrats, homosexuals, etc. Their sense of agency was strong but only insofar as they felt connected with the winds of history. Nazis were the strongest communitarians, feeling close ties for members of their own self-defined group but having little (if any regard) for those who fell outside their group.

I found that what drove such people is what I call the “Altruistic Perspective,” a particular way of looking at the world in which altruists see themselves as bound to others through their common humanity. Where the rest of us see a stranger, Those People see a fellow human being. This worldview is such a part of one of Those People's basic identity that it makes some actions—turning away from those in need, for example—simply unimaginable. It is analogous to the way we consider our options at a restaurant: It’s tough to get sushi at an Italian bistro; it’s just not on the menu. So it is with the Altruistic Perspective: It presents some of us with moral choices that differ significantly from the options available to others. Those people simply have a different cognitive menu. 

I think the first step is how to answer, "Do we say 'The law is the Law, even if you don't like it'. Practicing Judaism was illegal, if the Jews didn't want to be lawfully executed, that should have moved to someplace it was legal."

**Comment by natehn** [comment]
Thank you for sharing!
